data/net/minecraft/client/renderer/LevelRenderer.mapping

```diff
@@ -165,54 +177,54 @@ CLASS net/minecraft/client/renderer/LevelRenderer
 		ARG 2 pos
 	METHOD prepareCullFrustum (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/world/phys/Vec3;Lcom/mojang/math/Matrix4f;)V
 		ARG 1 poseStack
-		ARG 2 position
-		ARG 3 projection
+		ARG 2 cameraPos
+		ARG 3 matrix
```

Comment:
I believe the type of this matrix should be specified still whether projection or view.